### PR TITLE
Misc. testing cleanups

### PR DESCRIPTION
**Pull request by christarazi**
- config/testing: Add missing build tag for ginkgo
- test: Remove dead code for different Cilium namespaces

### CODE REVIEW

**Review by pchaigno — APPROVED** [review]

File: test/k8sT/DatapathConfiguration.go [hunk 1]
```diff
@@ -396,7 +395,7 @@ var _ = Describe("K8sDatapathConfig", func() {
 			res.ExpectSuccess()
 			tmpEchoPodPath = strings.Trim(res.Stdout(), "\n")
 			kubectl.ExecMiddle(fmt.Sprintf("sed 's/NODE_WITHOUT_CILIUM/%s/' %s > %s",
-				helpers.GetFirstNodeWithoutCilium(), echoPodPath, tmpEchoPodPath)).ExpectSuccess()
+				helpers.CiliumNamespace, echoPodPath, tmpEchoPodPath)).ExpectSuccess()
```
Comment (pchaigno) [inline test/k8sT/DatapathConfiguration.go]: The test failure is because of this line. I'm guessing it was replaced by mistake.

**Review by pchaigno — CHANGES_REQUESTED** [review]

File: test/k8sT/DatapathConfiguration.go (hunk 1 above)
Comment (christarazi) [reply test/k8sT/DatapathConfiguration.go]: Good catch, thanks!

**Review by pchaigno — APPROVED** [review]

**Pull request merged**